CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 ## [Unreleased]
 
+## [2.14.0] - 2018-09-27
+
+### Added
+- Credhub introduced as a user-accessible component (independent of future cf-deployment requirements)
+- Exposed SMTP variable to allow for account creation & password reset
+
+### Changed
+- One Kubernetes service per job now, which will require changes to role manifest YAML 
```

Comment:
More importantly, the service name has changed to include both the instance_group (was role) name, *and* the job name.

Reply:
Reworded to `One Kubernetes service per job now, whereby the service names will include both the instance group (previously the role) and job name, which impacts the role manifest YAML`